packages/integration-karma/test/events/focus-event-related-target/index.spec.js

```diff
@@ -1,7 +1,11 @@
 import { createElement } from 'lwc';
 import Container from 'x/container';
 
-it('should retarget relatedTarget', function () {
+// Note that focusin events are not fired in Chrome and Firefox if the browser content window is not focused.
+// Test for yourself: https://bl.ocks.org/nolanlawson/raw/d8ddc518041e9bf12498b8b50b39df95/
+// So this test may "flap" during local development if your browser window isn't focused (e.g. you're focused on
+// the DevTools instead).
+it('should retarget relatedTarget (note this test may fail if the browser window is not focused)', function () {
```

Comment:
ill leave just `should retarget relatedTarget`, the comment with note already explains the rest.

Reply:
I thought it might be helpful to see the note in the console as well, but yeah you're right, if they look at the test they'll see the comment.